src/hotspot/cpu/aarch64/atomic_aarch64.S

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021, Red Hat Inc. All rights reserved.
```

Comment:
Does this file work with the Windows assembler?

Reply:
I have no idea. If it doesn't, please tell me; I have nbo Windows system.

Reply:
I don't either unfortunately. Maybe @mo-beck or @luhenry could help? The Windows build on GitHub Actions failed cryptically with:

```
 Creating support/modules_libs/java.base/server/jvm.dll from 1045 file(s)
/usr/bin/bash: -c: line 0: syntax error near unexpected token `;'
make[3]: *** [lib/CompileJvm.gmk:143: /cygdrive/d/a/jdk/jdk/jdk/build/windows-aarch64/hotspot/variant-server/libjvm/objs/atomic_aarch64.obj] Error 2
```

Reply:
I suppose you could just move it to `os_cpu/linux_aarch64/` as these are only called from the Linux atomics?

Reply:
They're probably needed on Windows, or I'd have put them in linux_aarch64.

Reply:
I can confirm that assembly code targeted at Linux generally won't compile out of the box on Windows. For `windows_aarch64`, could we have simple fallbacks in C++ code that simply call `InterlockedAdd`, `InterlockedExchange`, and `InterlockedCompareExchange`? It would be similar to [atomic_windows_aarch64.hpp](https://github.com/openjdk/jdk/blob/master/src/hotspot/os_cpu/windows_aarch64/atomic_windows_aarch64.hpp)